pytest/tests/mocknet/local_test_node.py

```diff
@@ -92,6 +92,11 @@ def upload_neard_runner_config(self, config):
         # handled by local_test_setup_cmd()
         return
 
+    def run_cmd(self, cmd, raise_on_fail=False, return_on_fail=False):
+        logger.error(
+            "Does not make sense to run command on local host. The behaviour may not be the desired one."
```

Comment:
instead of saying "Does not make sense..." maybe just say "Please directly run the command on the local host..."?